fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/JDBCConstants.java

```diff
@@ -30,8 +30,8 @@
     public static final String CODE = "CODE";
     public static final String NUMBER_VALUE = "NUMBER_VALUE";
     public static final String QUANTITY_VALUE = "QUANTITY_VALUE";
-    public static final String QUANTITY_VALUE_LOW = "QUANTITY_VALUE_LOW";
-    public static final String QUANTITY_VALUE_HIGH = "QUANTITY_VALUE_HIGH";
+    public static final String _LOW = "_LOW";
+    public static final String _HIGH = "_HIGH";
```

Comment:
small suggestion here, LOW HIGH and match the common pattern for Java constants.